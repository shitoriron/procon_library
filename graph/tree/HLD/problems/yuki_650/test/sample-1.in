6
0 1
1 3
0 2
2 4
2 5
8
x 2 1 3 0 2
x 3 6 2 8 7
g 0 2
g 0 4
g 0 5
g 2 5
x 4 7 2 4 9
g 0 5
