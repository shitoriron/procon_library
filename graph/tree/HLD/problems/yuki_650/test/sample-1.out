1 3 0 2
30 23 16 14
1 3 0 2
1 0 0 1
19 29 8 18
